Todo/ViewController.swift

```diff
@@ -14,7 +14,7 @@ struct TodoViewModel {
     private let datasource = ["Walk dog", "Take out the trash", "Call Mom"]
     
     
-    func transformedDatasource() -> [String] {
+    func transformedDatasource(_ string:String) -> [String] {
```

Comment:
tid bit here...

for your params and types, the common structure is 

``` 
varName: Type
```

a space after the variable then the type so you'd have 

```
func transformedDatasource(_ string: String) -> [String] {
```